Tests/Magick.NET.Tests/Shared/Optimizers/ImageOptimizerTests.cs

```diff
@@ -448,6 +448,23 @@ public void Compress_CannotCompressPngFile_FileIsNotSmaller()
             });
         }
 
+        [TestMethod]
+        public void Compress_IgnoreUnsupportedFile_DoesNotThrowException()
+        {
+            var optimizer = new ImageOptimizer { IgnoreUnsupportedFormats = true };
+            Assert.IsFalse(optimizer.Compress(Files.InvitationTif));
+        }
+
+        [TestMethod]
+        public void Compress_IgnoreUnsupportedStream_DoesNotThrowException()
+        {
+            var optimizer = new ImageOptimizer { IgnoreUnsupportedFormats = true };
+            using (FileStream fileStream = OpenFile(Files.InvitationTif))
+            {
+                Assert.IsFalse(optimizer.Compress(fileStream));
```

Comment:
I think I would like to have the result of the compress method inside a variable first and assert that. But it seems I did not do that earlier. Probably because I had no idea what the name of the variable would be. Any ideas?

Reply:
We could call it `compressionSuccess`:

```
[TestMethod]
public void Compress_IgnoreUnsupportedStream_DoesNotThrowException()
{
    var optimizer = new ImageOptimizer { IgnoreUnsupportedFormats = true };
    using (FileStream fileStream = OpenFile(Files.InvitationTif))
    {
        var compressionSuccess = optimizer.Compress(fileStream);
        Assert.IsFalse(compressionSuccess);
    }
}
```

Reply:
Sound like a good name :)

Reply:
Could you do this in the PR where you refactor the tests?

Reply:
Sure thing I will include this change in the next PR